ingest/defaults/config.yaml

```diff
@@ -46,6 +46,7 @@ curate:
   # This is the first step in the pipeline, so any references to field names in the configs below should use the new field names
   field_map:
     accession: accession
+    accession_version: accession_version
```

Comment:
I chose the column name `accession_version` to be clearer, but open to changing it to match the existing pattern `accession_rev`.

Reply:
`Accession_version` is what NCBI actually calls it and what will be used by a similar project I'm working on, so I fully support `accession_version` - it's the right term!